pkgs/development/python-modules/pip/default.nix

```diff
@@ -16,26 +16,16 @@
 
 buildPythonPackage rec {
   pname = "pip";
-  version = "20.2.4";
+  version = "20.3";
   format = "other";
 
-  src = fetchFromGitHub {
-    owner = "pypa";
-    repo = pname;
-    rev = version;
-    sha256 = "eMVV4ftgV71HLQsSeaOchYlfaJVgzNrwUynn3SA1/Do=";
-    name = "${pname}-${version}-source";
+  src = fetchPypi {
+    inherit pname version;
+    sha256 = "03kfx8ymqllv5lyngnylqks6h3z04ky29l29kcm2vhpaarkcmrws";
   };
 
   nativeBuildInputs = [ bootstrapped-pip ];
 
-  patches = lib.optionals isPy27 [
-    (fetchpatch {
-      url = "https://github.com/pypa/pip/commit/94fbb6cf78c267bf7cdf83eeeb2536ad56cfe639.patch";
```

Comment:
This patch made it with the 20.3 release:
https://github.com/pypa/pip/pull/8666